common/app/model/liveblog/BodyBlock.scala

```diff
@@ -83,7 +88,8 @@ case class BodyBlock(
   publishedDate: Option[DateTime],
   lastModifiedDate: Option[DateTime],
   contributors: Seq[String],
-  elements: Seq[BlockElement]
+  elements: Seq[BlockElement],
+  pageElements: Seq[PageElement]
```

Comment:
it's not clear what the distinction between elements and pageElements is

Reply:
I'd still rename this dotcomRenderingPageElements or something just so it's clear that it's nothing to do with existing liveblog implementation